src/main/java/frc/robot/RobotContainer.java

```diff
@@ -18,12 +18,14 @@ public class RobotContainer {
     private Intake intake;
     private XboxController xboxController = new XboxController(0);
     private JoystickButton a = new JoystickButton(xboxController, XboxController.Button.kA.value);
+    private JoystickButton b = new JoystickButton(xboxController, XboxController.Button.kB.value);
+    private JoystickButton x = new JoystickButton(xboxController, XboxController.Button.kX.value);
+    private JoystickButton y = new JoystickButton(xboxController, XboxController.Button.kY.value);
```

Comment:
1. I don't think you should commit any code to shared files that isn't crucial. Otherwise, you'll have many merge conflicts when you merge all the subsystems. If you need something for testing, just don't commit it.
2. Just [FYI](https://dictionary.cambridge.org/dictionary/english/fyi), you should use [`CommandXboxController`](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj2/command/button/CommandXboxController.html) instead. That way you won't need to create buttons by yourself.

Reply:
5703c7fe03c04fb3a3f942d77b70f0869886b3a4